staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/roundtrip.go

```diff
@@ -350,14 +350,14 @@ func roundTrip(t *testing.T, scheme *runtime.Scheme, codec runtime.Codec, object
 	// decode (deserialize) the encoded data back into an object
 	obj2, err := runtime.Decode(codec, data)
 	if err != nil {
-		t.Errorf("%v: %v\nCodec: %#v\nData: %s\nSource: %#v", name, err, codec, dataAsString(data), dump.Pretty(object))
+		t.Errorf("%v: %v\nCodec: %#v\nData: %s\nSource: %s", name, err, codec, dataAsString(data), dump.Pretty(object))
 		panic("failed")
 	}
```

Comment:
1. Can you confirm that using %s instead of %#v will display the content properly? Could you tell me why if you can confirm that?
2. as if name is string too. Why not using %s instead of %v?

Reply:
> 1. Can you confirm that using %s instead of %#v will display the content properly? Could you tell me why if you can confirm that?

Sure, this is based on the fmt verb behavior list in https://pkg.go.dev/fmt#hdr-Printing. The verb %#v applied to a string represents the string as it would appear in Go source, which means it appears inside quotes and special characters like newlines are escaped.

>     2. as if name is string too. Why not using %s instead of %v?

There is no difference. %v uses the default format for the value's type, and the default format for string values is %s. Here is an example: https://go.dev/play/p/BbCeI9tNgfM.